bsc/rate.go

```diff
@@ -16,3 +16,10 @@ func ConvertBCAmountToBSCAmount(bcAmount int64) *big.Int {
 	bscAmount := sdk.NewInt(bcAmount).Mul(decimals)
 	return bscAmount.BigInt()
 }
+
+func ConvertBSCAmountToBCAmount(bscAmount *big.Int) int64 {
```

Comment:
Please add some comments here, in case of misuse of this function in the future. (it is only can be applied to BNB)

Reply:
Done